package.json

```diff
@@ -130,16 +130,17 @@
     "process": "^0.11.10",
     "puppeteer": "^24.1.1",
     "rimraf": "^6.0.1",
-    "rollup": "^4.32.1",
+    "rollup": "^4.34.2",
     "rollup-plugin-istanbul": "^5.0.0",
     "staged-git-files": "^1.3.0",
     "stylus": "^0.64.0",
     "testcafe": "^3.7.1",
     "testcafe-browser-provider-saucelabs": "^3.0.0",
-    "vitest": "^3.0.4",
+    "vitest": "^3.0.5",
     "whatwg-fetch": "^3.6.20"
   },
   "overrides": {
-    "eslint": "^9.19.0"
+    "eslint": "^9.19.0",
+    "strip-comments": "^2.0.1"
```

Comment:
We need this override in order to fix the library sandbox issue where the Tags library file cannot be created by the sandbox if the library modules contain ES6 code.